js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -419,17 +424,59 @@ const AssetGraphExplorerWithData = ({
       shortcutFilter={(e) => e.altKey && e.code === 'KeyE'}
     >
       {expandedGroups.length === 0 ? (
-        <Button
-          title="Expand all groups"
-          icon={<Icon name="unfold_more" />}
-          onClick={() => setExpandedGroups(allGroups)}
-        />
+        <Tooltip
+          content={
+            <>
+              Expand all groups{' '}
+              <Caption
+                style={{
+                  width: 16,
+                  borderRadius: 4,
+                  padding: '2px 4px',
+                  color: colorAccentWhite(),
+                  background: colorAccentGray(),
```

Comment:
Does the gray accent background look okay in dark mode too? Since this lives inside a tooltip, maybe it makes sense for it to be the same color in both modes?

Reply:
I thought it looked ok, but I think I should actually probably use `backgroundGray` since that's what we use in our tag component. will swap it out